components/deployment.sh

```diff
@@ -28,7 +28,7 @@ function itunes_connect() {
 
   if [[ $build_number ]]; then
     msg "Tagging build as $build_number"
-    git tag -a build-"$build_number" -m "Added by ios-tools"
+    git tag -a "build-$build_number" -m "Added by ios-tools"
```

Comment:
ok, i still don't understand "$VAR" syntax. does it actually put the " character in the command? or is it part of string interpolation?